crates/db_schema/src/lib.rs

```diff
@@ -59,6 +59,9 @@ pub enum SortType {
   TopAll,
   MostComments,
   NewComments,
+  TopHour,
+  TopSixHour,
```

Comment:
Nice job. We did need to make sure these got appended, to make sure people's existing sort types didn't get wiped out.